<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Assig11Class</class>
 <widget class="QMainWindow" name="Assig11Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assig11</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_button">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="update_button">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="sizeEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="colourEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="priceEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="quantityEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="linkEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>81</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Colour:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Quantity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Photo Link:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Available Coats</string>
    </property>
   </widget>
   <widget class="QListWidget" name="coatListWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>311</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="sortRadioButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort by Price</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="shuffleRadioButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Shuffle</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>382</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCoat_Shop">
    <property name="title">
     <string>Coat Shop</string>
    </property>
   </widget>
   <addaction name="menuCoat_Shop"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Assig11.qrc"/>
 </resources>
 <connections/>
</ui>
